### PR TITLE
Update README.md #6

### PR DESCRIPTION
**Pull request by obaidofj**
writing the README.md, which include these sections 

- Description About the website
- The file structure 
- a link goes to the design 
- a link for the GitHub page
- how to have the project in the local machine 
- and at the end our team member

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+# Github pages link:
+https://gsg-k1-fc.github.io/ecommerce-team4/.
+
+# File structure:
+files include the following files,
+- index.html file which containe html code.
+- script.js file which contain all javascript code.
+- style.css file which contain css style code.
+- assests folder which contain the images files.
+
```
Comment (sara219) [inline README.md]: We do it like that:

> ├── index.html
> └── style.css
> └── script.js
> assets

**Review by sarah-abu-zeineh — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
-# ecommerce-team4
\ No newline at end of file
+# Ecommerce Project - Team4
+
+This website includes two parts one for customers and one for sellers , the site shows 
+all the avilable items the customer can search items by name and can filter them by
+price or category, he can add items to cart , delete them from cart and see total price 
+of items in cart.
+the seller can add items edit them or delete them, and can search in his products.
```
Comment (sara219) [inline README.md]: the description is good, but how about we talk a little about the e-commerce site, ex: 

```
e-commerce is a responsive online shopping website, where the customer can see all the
available products and search for a specific item ... etc

```

**talk with other members to decide a name for the website!**

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+# File structure:
+files include the following files,
+- index.html file which containe html code.
+- script.js file which contain all javascript code.
+- style.css file which contain css style code.
+- assests folder which contain the images files.
+
+# How to run the project:
+to run the project you need to download this reapo and opening the index.html file in the browser.
```
Comment (sara219) [inline README.md]: do it like that: 

- clone the repo into your local machine 
`git clone git@github.com:gsg-K1-FC/ecommerce-team4.git`

- Open the HTML on your browser

**Review by sara219 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (obaidofj) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (obaidofj) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (obaidofj) [reply README.md]: ok

File: README.md [hunk 4]
```diff
@@ -1 +1,29 @@
-# ecommerce-team4
\ No newline at end of file
+# Ecommerce Project - Team4
+
```
Comment (khaleelsiaj) [inline README.md:2]: you can add the Figma link to our design write here @_@

Comment (obaidofj) [reply README.md:2]: ok

**Review by sara219 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,25 +1,32 @@
...
+where the customer can see all the available products and search for a specific item
+also the customer can filter items by price or category, 
+and he can add items to cart , delete them from cart and see total price 
 of items in cart.
-the seller can add items edit them or delete them, and can search in his products.
+also the website has the seller part so the seller can add items ,edit them or delete them, 
+and can search in his items or products.
 
 # Design link:
+figma.com/file/VjvsWuXEHf4O00vzUOPON1/E-commerce?node-id=20%3A2
```
Comment (sarah-alsharif0) [inline README.md:12]: We haven't finished the design yet, please wait until tomorrow.

**Review by sarah-alsharif0 — CHANGES_REQUESTED** [review]

**Review by sarah-abu-zeineh — DISMISSED** [review]
The design doesn't complete to attach it here , so we can't added it before we done everything in it  , you can see more details in our channel in discord   .

**Review by sarah-abu-zeineh — APPROVED** [review]

**Review by sarah-alsharif0 — APPROVED** [review]

**Review by khaleelsiaj — APPROVED** [review]

**Pull request merged**